fancygit-completion

```diff
@@ -30,8 +30,8 @@ _fancygit() {
         --enable-host-name \
         --disable-host-name \
         --enable-user-symbol \
-        --disable-user-symbol \
-        --set-user-name \
```

Comment:
Ooops. @szczys I think we should keep those commands too.

Reply:
:facepalm: terribly sorry for this error. I've corrected that change to add the desired completions without removing anything else.